test/functional/inputs_test.rb

```diff
@@ -42,6 +42,13 @@
       line = lines.detect { |l| l.include? "--attrs" }
       line.wont_be_nil
     end
+
+    it "includes the --input option" do
+      result = run_inspec_process("exec help", lock: true) # --no-create-lockfile option breaks usage help
+      lines = result.stdout.lines
+      line = lines.detect { |l| l.include? "--input" }
```

Comment:
`assert_include lines.join, "--input"`

but both versions have the problem that `--input-file` will match this... so probably `assert_match` w/ `$` or something.